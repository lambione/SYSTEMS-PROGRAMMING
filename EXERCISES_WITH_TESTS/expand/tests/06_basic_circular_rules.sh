#!/bin/bash
exec "$1" tests/circular_rules.txt <<EOF
hello my friend!
"ciao" means "hello"
"ciao" also means "goodbye"
EOF
