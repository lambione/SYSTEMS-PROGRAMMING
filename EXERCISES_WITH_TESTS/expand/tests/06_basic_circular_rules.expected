ciao my friend!
"hello" means "ciao"
"hello" also means "goodbye"
